static/styles/style.css

```diff
@@ -1,25 +1,47 @@
 @font-face {
-  font-family: "Pricedown";
-  src: url("../fonts/pricedown.eot");
+  font-family: "pricedown";
+  src: url("fonts/pricedown.eot");
```

Comment:
Throughout this, I notice you've removed the `../` before the directory. This makes Flask not be able to find these files and folders. Please re-add everything that you've removed that I personally added to remedy this.

Reply:
Okay, you need to tell me when you change the code because I use my personal copy which is on my PC. I do not clone it to my PC.

Reply:
You should re-clone (or pull) when there are changes that have been made. Please do that from now on to avoid conflicts like these. These changes *will* break your method of hosting the demo site, but you *can* create another local copy from the up-to-date source and fix things so that you can host. This is just how collaboration works for the time being.